theme/static/css/hydroshare_core.css

```diff
@@ -1406,7 +1406,7 @@ body > .container{
     word-break:break-word;
 }
 
-#btn-download-all, #btn-add-file{
+#btn-download-all, #btn-add-file, #btn-signin-irods, #btn-select-irods-file {
```

Comment:
I agree a common class would be better, but I will not do it this time around since there are too many changes already in this branch, and I don't want to introduce more nice-to-have changes to minimize potential issues. Let's leave it as is now at least for this feature branch to be merged in.